libs/application/templates/parental-leave/src/fields/Review/index.tsx

```diff
@@ -171,231 +148,456 @@ const Review: FC<ReviewScreenProps> = ({
-                        s as ValidOtherParentAnswer,
-                      )
-                    }}
-                  />
-                ) : (
-                  <RadioValue>
-                    {
-                      getValueViaPath(
-                        application.answers,
-                        'otherParent',
-                      ) as string[]
-                    }
-                  </RadioValue>
-                )}
-              </Box>
-              {statefulOtherParentConfirmed === 'manual' && (
-                <>
-                  <Box marginTop={3} />
-                  <GridRow>
-                    <GridColumn span="6/12">
-                      {editable ? (
-                        <Input
-                          id="otherParentName"
-                          name="otherParentName"
-                          label={formatMessage(
-                            parentalLeaveFormMessages.shared.otherParentName,
-                          )}
-                          ref={register}
-                        />
-                      ) : (
-                        <Text>
-                          {
-                            getValueViaPath(
-                              application.answers,
-                              'otherParentName',
-                            ) as string[]
-                          }
-                        </Text>
-                      )}
-                    </GridColumn>
-                    <GridColumn span="6/12">
-                      {editable ? (
-                        <Input
-                          id="otherParentId"
-                          name="otherParentId"
-                          label={formatMessage(
-                            parentalLeaveFormMessages.shared.otherParentID,
-                          )}
-                          ref={register}
-                        />
-                      ) : (
-                        <Text>
-                          {
-                            getValueViaPath(
-                              application.answers,
-                              'otherParentId',
-                            ) as string[]
-                          }
-                        </Text>
-                      )}
-                    </GridColumn>
-                  </GridRow>
-                </>
-              )}
-            </Box>
-          </AccordionItem>
-
-          <AccordionItem
-            id="id_3"
-            label={formatMessage(
-              parentalLeaveFormMessages.shared.paymentInformationSubSection,
-            )}
-            startExpanded={allItemsExpanded}
-          >
-            <Box paddingY={4}>
-              <GridRow>
-                <GridColumn span="6/12">
-                  {editable ? (
-                    <Input
-                      id="payments.bank"
-                      name="payments.bank"
+    <>
+      <ReviewGroup
+        isEditable={editable}
```

Comment:
Here we need to keep in mind that only a primary parent can edit the other parent